linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/names/WebDelegator.scala

```diff
@@ -168,7 +168,7 @@ private[admin] class WebDelegator(
   def apply(req: Request): Future[Response] = req.method match {
     case Method.Get => (req.params.get("d"), req.params.get("n")) match {
       case (Some(DtabStr(dtab)), Some(PathStr(path))) =>
-        val DstBindingFactory.Namer(namer) = linker.params[DstBindingFactory.Namer]
+        val DstBindingFactory.Namer(namer) = linker.routers.head.params[DstBindingFactory.Namer]
```

Comment:
Is this right? Is it ok to resolve the dtab of one router with the name interpreter of another? Are all router nameinterpreters the same?